kong/db/migrations/core/007_140_to_200.lua

```diff
@@ -0,0 +1,38 @@
+return {
+  postgres = {
+    up = [[]],
+
+    teardown = function(connector)
+      assert(connector:query([[
+        DO $$
+        BEGIN
+          ALTER TABLE IF EXISTS ONLY "plugins" DROP COLUMN "run_on";
```

Comment:
Do we also need to `DROP` indexes just to be explicit?

Reply:
Ok, I am fine with this, even though with other migrations we have been explicit. Though I guess there are already multiple styles.